server/api/routeImages.js

```diff
@@ -35,13 +35,25 @@ router.post('/', (req, res, next) => {
         return res.status(500).send(err);
       }
       //if no error,add the image name and send the file name and path back to the client
-      RouteImage.create({
-        fileName: file.name,
-        filePath: `/uploads/${file.name}`,
-        userId: req.user.id,
-      }).then(() => {
-        res.send({ fileName: file.name, filePath: `/uploads/${file.name}` });
+      ClimbingRoute.create({
```

Comment:
We actually don't want to create a new climbing route every time some1 uploads a video. We just need to associate the new image with a climbing route and id. So you're creating a new entry in the  route image table and giving it a value for climbingRouteId and userId

Reply:
Also shouldn't the file path be `./assets/uploads/${file.name}`  ?

Reply:
1. Yeah i will make the fix for the climbing route, thanks for the clarification !

2. The file path for the image when it is uploaded is **./public/uploads/${file.name}**  

Essentially would you want the image to be uploaded into the assets section as it's own separate directory instead, like  **./public/assets/uploads/${file.name}**   ?

Reply:
@nries1  or did you mean the file path/url to be: **`./public/uploads/${file.name}`**